test/unittests/execute_control_test.cpp

```diff
@@ -178,6 +178,7 @@ TEST(execute_control, nested_blocks)
             i32.const 4
             i32.or
             set_local 0
+            i32.const 1
```

Comment:
What is the reason for this? To make the block arity 1 and support jumping to the func?

Reply:
This is for `br 3` (aka `return`) to work, because the function result has type `[i32]`.

Reply:
Yeah, just wanted to confirm I understand it properly.